src/App/_tests_/ForgotPassword.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from '../../enzyme';
+import ForgotPassword from '../Components/PasswordReset/ForgotPassword';
+import mockAxios from '../../__mocks__/axios';
+
+const forgotPassword = shallow(<ForgotPassword />);
+
+describe('ForgotPassword', () => {
+  it('renders the ForgotPassword component', () => {
+    expect(forgotPassword.find('div').exists()).toBe(true);
```

Comment:
would you test more than the div itself to exists??

Reply:
noted @MCFrank16